docs/topics/Gateway.md

```diff
@@ -191,6 +300,9 @@ For larger bots, client state can grow to be quite large. We recommend only stor
 
 ## Guild Subscriptions
 
+> info
+> While not deprecated, Guild Subscriptions have been superceded by [Gateway Intents](#DOCS_TOPICS_GATEWAY/gateway-intents). You may continue to use guild subscriptions, but we recommend migrating to intents for even better results.
```

Comment:
```suggestion
> While not deprecated, [Guild Subscriptions](#DOCS_TOPIC_GATEWAY/guild-subscriptions) have been superceded by [Gateway Intents](#DOCS_TOPICS_GATEWAY/gateway-intents). You may continue to use guild subscriptions, but we recommend migrating to intents for even better results.
```

Reply:
why are you linking to the title that is right above that paragraph? That seems overly pointless